scripts/check-node-version.mjs

```diff
@@ -0,0 +1,33 @@
+import chalk from 'chalk';
+import pkg from '../package.json' assert { type: 'json' };
+
+const requiredNodeVersion = pkg.engines?.node;
+const currentVersion = process.versions.node;
+
+const requiredNodeVersionNumStr = requiredNodeVersion.replace(/[^\d.]/g, '');
+
+const requiredNodeVersionWholeNumber = Math.floor(
+    parseFloat(requiredNodeVersionNumStr)
+);
+const currentVersionWholeNumber = Math.floor(parseFloat(currentVersion));
```

Comment:
I'll LGTM this but i'm gonna refactor this to use https://github.com/npm/node-semver

Reply:
Ah yea. Good call. I can just leave this open then?